src/components/Layout.js

```diff
@@ -90,9 +91,7 @@ const Layout = (props) => {
 
     if (props.location.hash && !props.location.hash.includes("gatsby")) {
       const idTag = props.location.hash.split("#")
-      if (document.getElementById(idTag[1]) !== null) {
-        document.getElementById(idTag[1]).scrollIntoView(false)
```

Comment:
The problem we had. `scrollIntoView(false)` will put the anchor at the bottom of the viewport.